_submissions/round-12/0/2015-02-10-about-pandurang-kolekar.md

```diff
@@ -0,0 +1,28 @@
+---
+date: 2014-02-10
+round: Round 12
+title: About Pandurang Kolekar
+author: Pandurang Kolekar
+permalink: /2015/02/pandurang-kolekar/
+tags:
+  - Biography
+---
+
+#Pandurang Kolekar
```

Comment:
Please remove this line - the title will appear automatically based on the 'title' field in the header.